app/models/Event.js

```diff
@@ -47,27 +47,25 @@ const eventSchema = new Schema({
       }
     }
   },
-  rsvp: {
-    yes: {
-      type: Schema.Types.ObjectId,
-      ref: 'User'
-    },
-    no: {
-      type: Schema.Types.ObjectId,
-      ref: 'User'
-    },
-    mayBe: {
-      type: Schema.Types.ObjectId,
-      ref: 'User'
-    }
-  },
+
+  rsvpYes: [{
+    type: Schema.Types.ObjectId,
+    ref: 'User'
+  }],
+  rsvpMaybe: [{
+    type: Schema.Types.ObjectId,
+    ref: 'User'
+  }],
+  rsvpNo: [{
+    type: Schema.Types.ObjectId,
+    ref: 'User'
+  }],
   slots: {
-    type: Number,
-    required: true,
+    type: String,
```

Comment:
we can have slots as number,if we do this +1 then it will append 1 in string.

Reply:
sir, when we are changing the slots it will already be updated by the API and also i have added validation of isNumeric so that user can enter only numerical as a string

Reply:
isEmpty validation would be fine

Reply:
sir for the validation of npm directory , input has to be a string 
if i need to keep the validation then i have to make the type as string
so , what do u suggest , keep the type as string and use validation
or make it Number and remove all the validations?

Reply:
ok we can have string then